pyanaconda/ui/cockpit/anaconda-webui/src/timezone/TimeDate.jsx

```diff
@@ -15,15 +15,34 @@
  * along with This program; If not, see <http://www.gnu.org/licenses/>.
  */
 import cockpit from 'cockpit';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
 import {
-    PageSection
+    Form, FormGroup,
+    PageSection,
+    Switch,
 } from '@patternfly/react-core';
 
 import { Header } from '../Common.jsx';
 
+// This is a wrapper around timedatectl dbus API
```

Comment:
This should be fine to use now, but we might eventually add a fallback for some minimal environments where timedatectl might not be present as IIRC @poncovka or @jkonecny12 mentioned. :)

Reply:
indeed, i remember this talk about system bus not being available as well. let's use this however, so that people have some data to play with tomorrow.

Reply:
Honestly, I'm starting to be convinced that everything this UI is using should be on our DBus to avoid having logic in UI which we inclined in the past and why we have the current ui code so heavy and hard to change. However, not important right now.